src/data/faq.json

```diff
@@ -1879,6 +1879,67 @@
                 "title": "Risk Assessment",
                 "id": "risk_assessment",
                 "accordion": [
+                    {
+                        "title": "Why do I receive a risk warning several days after the risk encounter?",
+                        "anchor": "risk_encounter_not_realtime",
+                        "active": false,
+                        "textblock": [
+                            "The Corona-Warn-App does not provide real-time warning functionality. The time at which a risk warning is sent to the contacts of the last 14 days depends on when the positive test result was shared via the app. Only after a user has voluntarily shared a positive test result, other users can be warned via the CWA about an encounter which occurred before that time. With this warning, contacts receive an estimation of their personal risk.",
+                            "See FAQ: <ul><li><a href='#risk_encounter_no_warning' target='_blank'>Why don't I get a risk encounter warning even though I've had contact with someone who has a risk encounter displayed in their app?</a></li><li><a href='#risk_encounter_different_devices' target='_blank'>How do differences in risk assessment occur on multiple devices, even if the respective smartphones were not far apart?</a></li><li><a href='#pos_test_no_risk_encounter' target='_blank'>I tested positive. Why does a person in the same household not receive a risk warning?</a></li><li><a href='#risk_encounter_doubted' target='_blank'>Why do I receive a risk warning for a date when I had no contacts / did not leave the house?</a></li><li><a href='#risk_encounter_place' target='_blank'>Are people and/or places assigned to a risk encounter?</a></li></ul>"
+                        ]
+                    },
+                    {
+                        "title": "Why don't I get a risk encounter warning even though I've had contact with someone who has a risk encounter displayed in their app?",
+                        "anchor": "risk_encounter_no_warning",
+                        "active": false,
+                        "textblock": [
+                            "Risk warnings are only enabled by sharing positive test results via the Corona-Warn-App. Displaying a risk encounter on one device does not result in a risk warning on other users' devices. Only if a user also shares a positive test result via the CWA a warning can be issued to their contacts.",

```

Comment:
"Only if a user also shares a positive test result via the CWA a warning can be issued ..." =>

"Only if a user also shares a positive test result via the CWA can a warning be issued ..."

word order

Reply:
Changed it in https://github.com/corona-warn-app/cwa-website/pull/2529/commits/e25bfbeb50abbc2054e6fed0564c3cd2e725c5c9, thanks!